src/librustdoc/html/static/main.js

```diff
@@ -2716,10 +2716,6 @@ function defocusSearchBar() {
         };
     }
 
-    window.onresize = function() {
-        hideSidebar();
-    };
```

Comment:
This will impact desktops too, right? Is that intentional? Do you know why this was added in the first place?

Reply:
It won't have impact on desktop because it was only used for mobile devices. As for what it was used originally, I'm not sure... It was three years ago so things evolved quite a lot in the meantime in both rustdoc and mobile web browsers...

Reply:
Ok, seems reasonable.